programs/clearing_house/src/math/margin.rs

```diff
@@ -54,23 +54,16 @@ pub fn calculate_size_premium_liability_weight(
     let size_sqrt = ((size * 10) + 1).nth_root(2); //1e9 -> 1e10 -> 1e5
 
     let liability_weight_numerator = liability_weight
-        .checked_sub(
-            liability_weight
-                .checked_div(max(1, SPOT_IMF_PRECISION / imf_factor))
-                .ok_or_else(math_error!())?,
-        )
-        .ok_or_else(math_error!())?;
+        .safe_sub(liability_weight.safe_div(max(1, SPOT_IMF_PRECISION / imf_factor))?)?;
 
     // increases
-    let size_premium_liability_weight = liability_weight_numerator
-        .checked_add(
-            size_sqrt // 1e5
-                .checked_mul(imf_factor)
-                .ok_or_else(math_error!())?
-                .checked_div(100_000 * SPOT_IMF_PRECISION / precision) // 1e5 * 1e2
-                .ok_or_else(math_error!())?,
-        )
-        .ok_or_else(math_error!())?;
+    let size_premium_liability_weight = liability_weight_numerator.safe_add(
```

Comment:
this formatting is v strange lol

Reply:
i think its cuz of the comments, lemme see if i can change it